code/modules/mob/living/silicon/mommi/mommi_movement.dm

```diff
@@ -2,11 +2,14 @@
 /mob/living/silicon/robot/mommi/Process_Spaceslipping(var/prob_slip=5)
 	return 0
 
-/mob/living/silicon/robot/mommi/Process_Spacemove()
+/mob/living/silicon/robot/mommi/Process_Spacemove(var/check_drift = 0)
```

Comment:
Isn't this proc equal to the parent? Am I missing something?

Reply:
Yeah I misread something. Changing it.